samples/ExampleGallery/UAP/CameraEffectExample.xaml.cs

```diff
@@ -137,24 +146,35 @@ private void EffectCombo_SelectionChanged(object sender, SelectionChangedEventAr
         {
             await ResetEffectsAsync();
 
+            blurAmountSlider.Visibility = Visibility.Collapsed;
+            effectPropertySet = new PropertySet();
             string typeName = null;
 
             switch (effect)
             {
-                case displacementEffect: typeName = typeof(DisplacementEffect).FullName; break;
-                case rotatingTilesEffect: typeName = typeof(RotatedTilesEffect).FullName; break;
+                case displacementEffect:
+                    typeName = typeof(DisplacementEffect).FullName;
+                    break;
+                case rotatingTilesEffect:
+                    typeName = typeof(RotatedTilesEffect).FullName;
+                    break;
+                case gaussianBlurEffect:
+                    typeName = typeof(DynamicBlurVideoEffect).FullName;
+                    effectPropertySet["BlurAmount"] = blurAmountSlider.Value;
+                    blurAmountSlider.Visibility = Visibility.Visible;
```

Comment:
In the case the BlurEffect is chosen; set the initial value of the BlurAmount and show the Slider control in the UI.